protocol/v2/ssv/spectest/committee_msg_processing_type.go

```diff
@@ -45,7 +44,9 @@ func (test *CommitteeSpecTest) RunAsPartOfMultiTest(t *testing.T) {
 	lastErr := test.runPreTesting(logger)
 
 	if len(test.ExpectedError) != 0 {
-		require.EqualError(t, lastErr, test.ExpectedError)
+		// Don't want to compare error messages exactly because it's an implementation detail that
+		// might differ from the spec.
+		require.Error(t, lastErr)
 	} else {
 		require.NoError(t, lastErr)
 	}
```

Comment:
Seems like we **should be able to update error messages** (to add additional info in there, and make it more detailed if necessary) even if they diverge from those we have in spec ?

Could update spec messages instead, but seems like it's an implementation detail not worth spending time on, WDYT ?

Reply:
Actually even better approach here would probably be:
- to define a mapping (here in `ssv` repo) `testName`->`ExpectedErrorOverride`
- leave the check as it was (exact error string match) but take `ExpectedErrorOverride` value (instead of `ExpectedError`) if it exists in that mapping we added
- and then, for each error-message update like I have in this PR, we just have to manually verify each failing test (expected vs got error message) & override those errors where message differs now; checking for exact string match against `ExpectedErrorOverride` wont work though because error messages will have dynamic data in them (such as expected vs actual data and such), so regex-based matching should do the job

Reply:
@y0sher, @moshe-blox would be nice to get an ACK on this from one of you guys, and I think this PR is ready to merge